lib/src/main/scala/spinal/lib/bus/amba3/ahblite/AhbLite3SlaveFactory.scala

```diff
@@ -42,6 +42,13 @@ class AhbLite3SlaveFactory(bus: AhbLite3, incAddress: Int = 0) extends BusSlaveF
   override def readAddress()  = bus.HADDR
   override def writeAddress() = bus.HADDR
 
+  // TODO: Implement
```

Comment:
For implementation you can see https://github.com/SpinalHDL/SpinalHDL/pull/915/files#diff-12b66544be0890d1886c7938f0b2071a387c6b6279b1591d3d1bcb5640dde178R29-R40

Reply:
I was going to let you folks get that finalized before porting it back here.

Reply:
So @kleinai do you want to add it (you can use the example implementation in the link above) or delay implementation for a later PR?

Reply:
Done